### DISCUSSION TITLE
Launch command in background

### DISCUSSION THREAD

**Discussion by massot-c** [python-cmd2/cmd2#1218] [Q&A]
Hello guys,

I wander if cmd2 is implementing a mechanism to launch command in background.
I did not see anything like that in the documentation but mayby I missed something.

In my project, I would like to be able to launch command in background like we do in a shell with a '&' at the end of the command.
It would allows the user to continue its work using the CLI, while the command is running.

Thank you very much for this amazing repo.
Bye

**Comment by kmvanbrunt** [comment]
cmd2 does not have background commands built-in. In your command function, you will have to spawn another thread. I've used this approach in multiple cmd2 programs that I've written.
